### ISSUE TITLE
Review of Day Two

### ISSUE THREAD

**Issue by davclark** [dlab-berkeley/R-for-Data-Science#14]
Glad to see modern libraries covered. I think it's useful to show the figure from Hadley's dplyr workshop when you [introduce dplyr](https://github.com/dlab-berkeley/R-for-Data-Science/blame/2775db85850a1849188e0b60e9a268b36e6327c9/instructor/day_two.Rmd#L533). His paper with nice figures for this is [here](http://www.jstatsoft.org/article/view/v040i01).

It could also be nice to introduce tidyr, and I'd point people towards [reshape2](https://github.com/hadley/reshape), available on cran as `reshape2`.

**Comment by deniederhut** [comment]
Reshape2 is already in the day two materials - https://github.com/dlab-berkeley/R-for-Data-Science/blob/master/instructor/day_two.Rmd#most-tidying-can-be-done-with-two-r-packages.

I'm going to call this an enhancement request to add an image about split-apply-combine to the instructor notes. Or were you imagining that we would pull up the paper for the class?

**Comment by davclark** [comment]
As I mentioned in #13, it'd be nice to collect more complete reviews here. So, can we keep the review tag for now?

I think embedding the image is probably best, as it wouldn't entail a context shift from the slides. Here ya go:

![image](https://cloud.githubusercontent.com/assets/120710/12801137/118f508c-ca8f-11e5-9ba3-0b4199effc0e.png)

![image](https://cloud.githubusercontent.com/assets/120710/12801140/1df095de-ca8f-11e5-88f0-73990a567937.png)

![bitmoji](https://da8lb468m8h1w.cloudfront.net/v2/cpanel/8139922-128222025_3-s1-v1.png?palette=1&width=246)

**Comment by davclark** [comment]
Another note - you're introducing both plyr and dplyr. I know dplyr does less (used to anyway) - did you mean to introduce both? Sorry for the _ad hoc_ery. I'll do a more systematic review soon.

**Comment by deniederhut** [comment]
We don't present slides - or at least, we discourage instructors from using
slide decks. Ideally, the class is an interactive, live-coding session.
Typically, this means that all the students see is the RStudio IDE where
the instructor is "taking notes" in a script file and running the commands
to demonstrate output, errors, etc. So if you want to be 100% sure the
attendees see it, there will need to be a note about pulling up the image
and/or paper.

We used to have plyr, but I removed those materials to put dplyr in their
place... probably in December. I thought I took out all the references to
plyr, but I must have missed one (or more?).
